client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
Alright, that makes sense. I think we should keep the case_id since we get some ordering information from that and what order they saw each thing. Also, at some point, we're probably going to be encouraging students to add to the test files, so this won't be just a theoretical problem then.

I think a unique ID should be a hash of the question contents, the assignment name (e.g. 'cal/CS61A/sp15/hog'), and the test_name (e.g. 'Question 0'). All of this info is pretty easily available so hashing it shouldn't be hard. This should uniquely identify the question while allowing reasonable edits for the student (e.g. adding a test at the beginning, etc). What do you think?